monitor/requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+black==20.8b1
+flake8==3.8.4
+mypy==0.800
+pre-commit==2.14.0
```

Comment:
huh, I guess we should think about having a set of repo-wide tools because technically pre-commit is repo wide. Right now, pre-commit will not run on files in this folder. Only bridge.